cmake/FindCybSDK.cmake

```diff
@@ -1,86 +1,67 @@
-# Copyright 2018 Istituto Italiano di Tecnologia (IIT)
-# @author Mohamed Babiker Mohamed Elobaid <mohamed.elobaid@iit.it>
+# Copyright 2020 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
 
-# Finds the Cyb SDK
+# Finds the Sense Glove SDK
 #
 # This will define the following variables::
 #
-#   CybSDK_FOUND    - True if the system has the CybSDK 
-#   CybSDK_VERSION  - The version of the CybSDK  which was found
-#
-#
+#   CybSDK_FOUND         - True if the system has the Sense Glove SDK
+#   CybSDK_INCLUDE_DIR  - The include directory of the SDK
+#   CybSDK_LIBRARIES     - The name of the libraries to link against.
+###############################
 
-##### Utility #####
+# Check Directory of the CybSDK_DIR
+if(NOT DEFINED ENV{CybSDK_DIR})
+  message( FATAL_ERROR "Environment variable {CybSDK_DIR} is not defined: $ENV{CybSDK_DIR}" )
+else()
+  message("Environment variable {CybSDK_DIR}: $ENV{CybSDK_DIR}" )
+endif()
 
-# Check Directory Macro
-macro(CHECK_DIR _DIR)
-  if(NOT EXISTS "${${_DIR}}")
-    message(WARNING "Directory \"${${_DIR}}\" not found.")
-    set(CybSDK_FOUND FALSE)
-    unset(_DIR)
-  endif()
-endmacro()
+if(NOT DEFINED ENV{CybSDK_INCLUDE_DIRS})
```

Comment:
I would suggest not requesting the definition of multiple variables from the user. If the structure on the SDK is known, we can just "hardcode" the relative paths starting from  ``CybSDK_DIR``